docs/en/annotations.rst

```diff
@@ -106,7 +107,23 @@ is very important during development. If you don't set it to true you have to de
 This gives faster performance, however should only be used in production, because of its inconvenience
 during development.
 
-You can also use one of the ``Doctrine\Common\Cache\Cache`` cache implementations to cache the annotations:
+You can use a PSR-16 cache implementation to cache the annotations:
+
+.. code-block:: php
+
+    use Doctrine\Common\Annotations\AnnotationReader;
+    use Doctrine\Common\Annotations\CachedReader;
+    use Doctrine\Common\Cache\ApcCache;
```

Comment:
give an example using ApcuCache instead, as ApcCache is deprecated (due to using the legacy extension)

Reply:
btw, this use statement is actually unused here.